src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/UsersTest.kt

```diff
@@ -515,6 +515,57 @@ class UsersTest : InitialiseDatabasePerClassTestBase() {
       }
     }
 
+    @ParameterizedTest
+    @EnumSource(value = UserRole::class, names = ["CAS1_ADMIN", "CAS1_WORKFLOW_MANAGER", "CAS1_JANITOR", "CAS1_USER_MANAGER"])
+    fun `GET to users with an approved role returns list filtered by user's CRU management area`(role: UserRole) {
+      `Given a User`(roles = listOf(UserRole.CAS1_MATCHER)) { matcher, _ ->
+        `Given a User`(roles = listOf(UserRole.CAS1_MANAGER)) { manager, _ ->
+          `Given a User` { userWithNoRole, _ ->
+            `Given a User`(roles = listOf(role)) { requestUser, jwt ->
+              val apArea = `Given an AP Area`()
+
+              val probationRegionApArea = `Given an AP Area`()
+              val probationRegion = probationRegionEntityFactory.produceAndPersist {
+                withApArea(probationRegionApArea)
+              }
+
+              val userOne = userEntityFactory.produceAndPersist {
+                withProbationRegion(probationRegion)
+                withApArea(apArea)
+                withCruManagementArea(apArea.defaultCruManagementArea)
+              }
+
+              val userTwo = userEntityFactory.produceAndPersist {
+                withProbationRegion(probationRegion)
+                withApArea(apArea)
+                withCruManagementArea(apArea.defaultCruManagementArea)
+              }
```

Comment:
can we please have another user not in the cru management area we're filtering on?

Reply:
Added another user in the same apArea but different CRU management area.  Test still passes as expected.